stock_move_line_lock_qty_done/models/stock_move_line.py

```diff
@@ -3,13 +3,18 @@
 
 from odoo import _, models
 from odoo.exceptions import UserError
+from odoo.tools import config
 
 
 class StockMoveLine(models.Model):
 
     _inherit = "stock.move.line"
 
     def _check_qty_done_change_allowed(self, vals):
+        if config["test_enable"] and not self.env.context.get(
```

Comment:
I know this is an easy workaround that flourishes everywhere but it's not a good practice.

Introduce configuration setting that allows to enable or disable the feature.

Thanks !

Reply:
@rousseldenis I thought you might say so. ;)